javascript/javascript2/week3/fastestPresser.js

```diff
@@ -0,0 +1,70 @@
+const startButton = document.getElementById("game-starter");
+const secondsInput = document.getElementById("seconds-input");
+let countSLetter = document.getElementById("s-count-p")
+let countLLetter = document.getElementById("l-count-p")
+let sCounter = 0;
+let lCounter = 0;
+const sContainer = document.getElementById("s-container");
+const lContainer = document.getElementById("l-container");
+let youWonS = document.querySelector("#s-container h3")
+let youWonL = document.querySelector("#l-container h3")
+
+startButton.addEventListener("click", setTime)
+function setTime() { 
+    reset();
+    if (secondsInput.value >= 1) {
+        let timeInMs = secondsInput.value * 1000;
+        window.addEventListener("keydown", counter)
+        setTimeout(function () { winner() }, timeInMs)
+    } else {
+        alert("Invalid input: please set seconds first(must be equal or greater than 1).")
+    }
+}
+function counter(event) {
+    if (event.key === "s") {
+        sCounter++;
+        countSLetter.innerHTML = sCounter;
+    }
+    if (event.key === "l") {
```

Comment:
This can be an else if
```suggestion
    } else if (event.key === "l") {
```